bokeh/resources.py

```diff
@@ -33,25 +33,36 @@ def _cdn_base_url():
 def _get_cdn_urls(version=None, minified=True):
     if version is None:
         version = __version__.split('-')[0]
-    min = ".min" if minified else ""
+
+    # check the 'dev' fingerprint
+    dev = version.split('.')[-2]
+    # check if we want minified js and css
+    _min = ".min" if minified else ""
+
     base_url = _cdn_base_url()
+    rel_container = 'bokeh/release'
+    dev_container = 'bokeh/dev'
+    container = rel_container if dev != 'dev' else dev_container
+
     result = {
-        'js_files'  : ['%s/bokeh-%s%s.js' % (base_url, version, min)],
-        'css_files' : ['%s/bokeh-%s%s.css' % (base_url, version, min)],
+        'js_files'  : ['%s/%s/bokeh-%s%s.js' % (base_url, container, version, _min)],
+        'css_files' : ['%s/%s/bokeh-%s%s.css' % (base_url, container, version,_min)],
         'messages'  : [],
     }
+
     if len(__version__.split('-')) > 1:
         result['messages'].append({
             "type" : "warn",
             "text" : "Requesting CDN BokehJS version '%s' from Bokeh development version '%s'. This configuration is unsupported and may not work!" % (version, __version__)
         })
```

Comment:
This check can go away, right? Now if someone is using a dev build they will get the URL to the dev CDN version?